docker/ci/entrypoint.sh

```diff
@@ -124,21 +124,19 @@ setup_tests() {
 run_units() {
 	shopt -s extglob
 	reset_dbs
-	execute "time bundle exec turbo_tests --verbose -n $JOBS --runtime-log spec/support/runtime-logs/turbo_runtime_units.log spec/!(features) modules/**/spec/!(features)"
+	execute "time bundle exec turbo_tests --verbose -n $JOBS --runtime-log spec/support/runtime-logs/turbo_runtime_units.log {,modules/*/}spec/!(features)"
```

Comment:
What's the leading comma before `modules`?

Reply:
It is for empty string, so `{,modules/*/}spec/!(features)` expands to `spec/!(features)` and `modules/*/spec/!(features)`, then expended glob takes over

```bash
ruby -e 'p ARGV' {,,foo,,bar,,}
# ["", "", "foo", "", "bar", "", ""]

ruby -e 'p ARGV' pre-{,,foo,,bar,,}-post
# ["pre--post", "pre--post", "pre-foo-post", "pre--post", "pre-bar-post", "pre--post", "pre--post"]

ruby -e 'p ARGV' {,,}Gemfile*
# ["Gemfile", "Gemfile.lock", "Gemfile.modules", "Gemfile", "Gemfile.lock", "Gemfile.modules", "Gemfile", "Gemfile.lock", "Gemfile.modules"]
```

Reply:
I find the current glob is simpler to parse through but this version is more succinct :)

Nice to learn about the `,`